ui/tests/integration/components/secret-list-header-test.js

```diff
@@ -37,4 +39,63 @@ module('Integration | Component | secret-list-header', function (hooks) {
       }
     }
   });
+
+  test('it should render new pki beta button and remain the same for other engines', async function (assert) {
+    const backends = supportedSecretBackends();
+    const numExpects = backends.length + 1;
+    assert.expect(numExpects);
+
+    this.server.post('/sys/capabilities-self', () => {});
+
+    for (const type of backends) {
+      const data = this.server.create('secret-engine', 2, { type });
+      this.model = mirageToModels(data);
+      await render(hbs`
+        <SecretListHeader
+          @model={{this.model}}
+        />
+        <div id="modal-wormhole"></div>
+      `);
+      const oldPkiBetaButtonSelector = '[data-test-old-pki-beta-button]';
+      const oldPkiBetaModalSelector = '[data-test-modal-background="New PKI Beta"]';
+
+      if (type === 'pki') {
```

Comment:
I'm surprised the linter let a conditional render here - I came across it a couple months ago 🤔

Reply:
Are conditionals usually not allowed in tests? I think the original test had a conditional

Reply:
I got a linting error when I had one a while ago! But if it's not coming up, then I wouldn't worry about it. It might have been for a specific way the conditional was written 🤷